tests/lib/bundle_cli_test.py

```diff
@@ -26,3 +26,9 @@ def test_collapse_bare_command_non_empty_str_args_with_escaped_char(self):
         expected_result = ['cl', 'run', "echo 'hello world!'"]
         actual_result = self.bundle_cli.collapse_bare_command(argv)
         self.assertEqual(actual_result, expected_result)
+
+    def test_collapse_bare_command_punctuation_char_semicolon(self):
+        argv = ['cl', 'run', '---', 'date', ';', 'sleep', '10']
+        expected_result = ['cl', 'run', 'date ; sleep 10']
```

Comment:
Have you tested the alternatives
`cl run --- date; sleep 10` and `cl run --- date;sleep 10` ?

Reply:
This use case will be handled via rest api side. 
```
stanford@Stanfords-MacBook-Pro deployment % python3
Python 3.7.7 (default, Mar 10 2020, 15:43:33) 
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import shlex
>>> command = 'cl run --- date;sleep 10'
>>> lexer = shlex.shlex(command, posix=True, punctuation_chars=True)
>>> lexer.escapedquotes = '\'"'
>>> list(lexer)
['cl', 'run', '---', 'date', ';', 'sleep', '10']
>>> command = 'cl run --- date; sleep 10'
>>> lexer = shlex.shlex(command, posix=True, punctuation_chars=True)
>>> lexer.escapedquotes = '\'"'
>>> list(lexer)
['cl', 'run', '---', 'date', ';', 'sleep', '10']
```